src/Makefile

```diff
@@ -284,6 +284,8 @@ endif
 ### 3.4 Bits
 ifeq ($(bits),64)
 	CXXFLAGS += -DIS_64BIT
+else
+	CXXFLAGS += -mstackrealign
```

Comment:
This is in the wrong place! This line should be conditioned on *both* ARCH being x86-32 specifically and compiler being gcc (including mingw).